src/home/buttons/camera-button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Grid } from '@material-ui/core'
+
+function cameras() {
```

Comment:
We should not use the `function` keyword, since we always used lambdas (and they're sexier too)

Reply:
done